pysal/explore/momepy/__init__.py

```diff
@@ -0,0 +1,12 @@
+import momepy.datasets as datasets
+from momepy.dimension import *
+from momepy.distribution import *
+from momepy.diversity import *
+from momepy.elements import *
+from momepy.graph import *
+from momepy.intensity import *
+from momepy.shape import *
+from momepy.utils import *
+from momepy.weights import *
+from momepy.preprocessing import *
+from momepy.coins import *
```

Comment:
I think that this is not necessary. `momepy` exposes all public function and classes in the main namespace, so it may be easier (and future-proof) to just

```py
from momepy import *
```